servlet/5.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Servlet 5.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Servlet 5.0"
+date: 2020-07-20
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
 
 * [Jakarta Servlet 5.0 Release Record](https://projects.eclipse.org/projects/ee4j.servlet/releases/5.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0-SNAPSHOT.html) (HTML)
+* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0.pdf) (PDF)
+* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0.html) (HTML)
 * [Jakarta Servlet 5.0 Javadoc](./apidocs)
-* [Jakarta Servlet 5.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Servlet 5.0 TCK](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/jakarta-servlet-tck-5.0.0.zip)([sig](),[sha](),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.servlet:jakarta.servlet-api:jar:5.0.0-M1](https://search.maven.org/artifact/jakarta.servlet/jakarta.servlet-api/5.0.0-M1/jar)
+  * [jakarta.servlet:jakarta.servlet-api:jar:5.0.0](https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/servlet/jakarta.servlet-api/5.0.0/jakarta.servlet-api-5.0.0.jar)
```

Comment:
This should be the "final" maven coordinates for the API.  Along the lines of https://search.maven.org/artifact/jakarta.servlet/jakarta.servlet-api/5.0.0/jar

Reply:
Yes, I know this too ;) Still waiting for any +1 from the Servlet committers. I hope anyone will respond today.

Reply:
I've now put in the coordinates of the future link. It's of course not there yet.

Reply:
Thanks.  Again, I would leave this in a staging repo until the ballot is complete.  The promotion of the maven central artifacts is left up to the project team.